src/nvim/CMakeLists.txt

```diff
@@ -951,6 +951,7 @@ add_custom_command(
     ${PROJECT_SOURCE_DIR}/src/nvim/eval.lua
     ${PROJECT_SOURCE_DIR}/src/nvim/options.lua
     ${PROJECT_SOURCE_DIR}/src/nvim/vvars.lua
+    ${NVIM_RUNTIME_DIR}/doc/builtin.txt
```

Comment:
Should other generated files also be added here then?

Reply:
Nah, I'm deleting this line. The whole thing needs to be rewritten.